x-pack/test/functional/services/ml/data_frame_analytics_table.ts

```diff
@@ -81,6 +81,14 @@ export function MachineLearningDataFrameAnalyticsTableProvider({ getService }: F
       await testSubjects.existOrFail('mlAnalyticsJobViewButton');
     }
 
+    public async openEditFlyout() {
```

Comment:
We already have parts of this functionality in this service file, see methods `openRowActions` and `cloneJob` below. I'd suggest to make this one similar to `cloneJob`.

Reply:
Updated in 64abf306dfb6f2021e1536a2542b4631ba07de6a